src/test/coinstatsindex_tests.cpp

```diff
@@ -79,6 +79,10 @@ BOOST_FIXTURE_TEST_CASE(coinstatsindex_initial_sync, TestChain100Setup)
     // Shutdown sequence (c.f. Shutdown() in init.cpp)
     coin_stats_index.Stop();
 
+    // Make sure coin_stats_index is not used by the scheduler thread after it
+    // has been destroyed.
+    SyncWithValidationInterfaceQueue();
+
     // Rest of shutdown sequence and destructors happen in ~TestingSetup()
```

Comment:
Maybe remove this comment as well, or add it to the other places, to mirror fa176e253fb473767c61d4d8cd2d93e87d71a015?

Reply:
Removed.